File: 283-move-zeroes/move-zeroes.py
class Solution:
    def moveZeroes(self, nums: List[int]) -> None:
        """
        Do not return anything, modify nums in-place instead.
        """
        i = 0
        count=0
        j = len(nums)-1
        while i<=j:
            if nums[i]!=0:
                nums[count]=nums[i]  
                count+=1
            i+=1       

        while(count<=j):
            nums[count] = 0
            count+=1

File: 283-move-zeroes/move-zeroes.java
class Solution {
    public void moveZeroes(int[] arr) {
        int n  = arr.length;
         int count =0;
        for(int i =0;i<n;i++){
            if(arr[i]!=0){
                arr[count++] = arr[i];
            }
        }
         for(int i =count;i<n;i++){
                arr[i] = 0;
            
        }
    }
}